src/main/java/com/dataloom/datastore/search/controllers/SearchController.java

```diff
@@ -104,4 +104,18 @@ public String executeQueryJson( @RequestBody SearchRequest request ) {
     @Override public AuthorizationManager getAuthorizationManager() {
         return authorizations;
     }
+
+    @RequestMapping(
+            path = { ORGANIZATIONS },
+            method = RequestMethod.POST,
+            produces = { MediaType.APPLICATION_JSON_VALUE } )
+    @Override
+    public String executeOrganizationSearch( @RequestBody String searchTerm ) {
+        try {
+            return ObjectMappers.getJsonMapper().writeValueAsString( searchService.executeOrganizationKeywordSearch( searchTerm ) );
+        } catch ( JsonProcessingException e ) {
+            e.printStackTrace();
+        }
+        return "[]";
```

Comment:
We only use finally blocks if we need to clean something up such as a lock other resource not covered with try with resources.